packages/wdio-cli/src/supportedPackages.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * We have to use a string hash for value because InquirerJS default values do not work if we have
+ * objects as a `value` to be stored from the user's answers.
+ */
+export const supportedPackages = {
+    runner: [
+        { name: 'local', value: '@wdio/local-runner$--$local' }
+    ],
+    framework: [
+        { name: 'mocha', value: '@wdio/mocha-framework$--$mocha' },
+        { name: 'jasmine', value: '@wdio/jasmine-framework$--$jasmine' },
+        { name: 'cucumber', value: '@wdio/cucumber-framework$--$cucumeber' }
```

Comment:
why the `$--$` also 
```suggestion
        { name: 'cucumber', value: '@wdio/cucumber-framework$--$cucumber' }
```

Reply:
The idea was to make use of inquirer's question api to have the choices hold the object suggested in the issue. Thing is that Inquirer selects the 'default' option based on the value of the answer and not what is displayed.

For example:

We can have a collection of choices:

```js
const choices = [
  { name: 'Choice One', value : { shortName: 'choice1', foo: 'bar' }
]
```

And use this in the Inquirer questionnaire:

```js
const questions = [{
  type: 'checkbox',
  message: '',
  choices,
  default: [{ shortName: 'choice1', foo: 'bar' }]
}]
```

This wouldn't work, but by using the `$--$` to split the package name and the short name we can then just parse the answer value after the questionnaire is done to then use a more understandable object in the code.


-

I'll fix the typo.